worlds/minecraft/test/TestOptions.py

```diff
@@ -0,0 +1,54 @@
+from . import MCTestBase
+from ..Options import minecraft_options
+
+from BaseClasses import ItemClassification
+
+class AdvancementTestBase(MCTestBase):
+    options = {
+        "advancement_goal": minecraft_options["advancement_goal"].range_end
+    }
+
+    def test_beatable(self):
+        self.multiworld.state = self.multiworld.get_all_state(False)
+        self.assertBeatable(True)
+
+class ShardTestBase(MCTestBase):
+    options = {
+        "egg_shards_required": minecraft_options["egg_shards_required"].range_end,
+        "egg_shards_available": minecraft_options["egg_shards_available"].range_end
+    }
+
+    def test_beatable(self):
+        assert len(self.multiworld.get_unfilled_locations()) == len(self.multiworld.itempool)
+        self.multiworld.state = self.multiworld.get_all_state(False)
+        self.assertBeatable(True)
+
+class CompassTestBase(MCTestBase):
+    def test_compasses_in_pool(self):
```

Comment:
i feel like a better way to do this would be to declare your list and `for compass in compasses: assert compass in self.multiworld.itempool`